test/rpc_test.py

```diff
@@ -15,30 +15,74 @@
 # limitations under the License.
 #
 
+import time
 from helpers import unittest
 
 import luigi.rpc
-import server_test
```

Comment:
I had some difficulties with the sever_test. I'll amend this PR as per my [comment](https://github.com/spotify/luigi/pull/811#issuecomment-77155803).